README.md

```diff
@@ -371,6 +371,20 @@ if let foo = foo {
 }
 ```
 
+Or when unwrapping multiple optionals, prefer this:
+
+```swift
+if 
+    let foo = foo,
```

Comment:
I'm personally not a fan of `if\n` in any of our styling.  Is there a good reason to prefer this over having the first statement on the same line?